pkg/kubernetes/client/kubectl.go

```diff
@@ -76,6 +77,34 @@ func (k Kubectl) Namespaces() (map[string]bool, error) {
 	return namespaces, nil
 }
 
+type ErrNamespaceNotFound struct{}
+
+func (e ErrNamespaceNotFound) Error() string {
+	return "Namespace not found"
```

Comment:
Being a struct anyways, it should include the name of the offending namespace

Reply:
So we only detect the error type then swallow it - we don't actually report it. But, for potential future use-cases, this is worth doing.